test/verify/check-multi-machine

```diff
@@ -375,7 +375,7 @@ class TestMultiMachine(MachineCase):
         break_bridge(m2)
         self.login_and_go(None)
         b.go(machine_path)
-        with b.wait_timeout(120):
+        with b.wait_timeout(240):
```

Comment:
This feels very long for a test. Do we need to decrease some timeouts in Cockpit itself?